src/platform/common.h

```diff
@@ -31,6 +35,7 @@ struct AVHWFramesContext;
 struct AVCodecContext;
 struct AVDictionary;
 
+#ifdef _WIN32
 // Forward declarations of boost classes to avoid having to include boost headers
 // here, which results in issues with Windows.h and WinSock2.h include order.
 namespace boost {
```

Comment:
Is this still relevant for Windows with our current build tools?

Reply:
Yep it still is.

```
In file included from C:/msys64/ucrt64/include/boost/asio/detail/win_event.hpp:25,
                 from C:/msys64/ucrt64/include/boost/asio/detail/event.hpp:23,
                 from C:/msys64/ucrt64/include/boost/asio/execution/blocking_adaptation.hpp:19,
                 from C:/msys64/ucrt64/include/boost/asio/execution.hpp:22,
                 from C:/msys64/ucrt64/include/boost/asio/any_completion_executor.hpp:22,
                 from C:/msys64/ucrt64/include/boost/asio.hpp:20,
                 from C:/Users/camer/moonlight-src/LB_Sunshine/src/platform/common.h:14,
                 from C:\Users\camer\moonlight-src\LB_Sunshine\src\platform\windows\input.cpp:18:
C:/msys64/ucrt64/include/boost/asio/detail/socket_types.hpp:24:4: error: #error WinSock.h has already been included
   24 | #  error WinSock.h has already been included
      |    ^~~~~
In file included from C:/msys64/ucrt64/include/boost/asio/detail/socket_types.hpp:33:
C:/msys64/ucrt64/include/winsock2.h:15:2: warning: #warning Please include winsock2.h before windows.h [-Wcpp]
   15 | #warning Please include winsock2.h before windows.h
      |  ^~~~~~~
In file included from C:/msys64/ucrt64/include/windows.h:72,
                 from C:/msys64/ucrt64/include/winsock2.h:23,
                 from C:/msys64/ucrt64/include/boost/asio/detail/socket_types.hpp:33,
                 from C:/msys64/ucrt64/include/boost/asio/detail/win_event.hpp:25,
                 from C:/msys64/ucrt64/include/boost/asio/detail/event.hpp:23,
                 from C:/msys64/ucrt64/include/boost/asio/execution/blocking_adaptation.hpp:19,
                 from C:/msys64/ucrt64/include/boost/asio/execution.hpp:22,
                 from C:/msys64/ucrt64/include/boost/asio/any_completion_executor.hpp:22,
                 from C:/msys64/ucrt64/include/boost/asio.hpp:20,
                 from C:\Users\camer\moonlight-src\LB_Sunshine\src\platform/common.h:14,
                 from C:\Users\camer\moonlight-src\LB_Sunshine\src\input.h:9,
                 from C:\Users\camer\moonlight-src\LB_Sunshine\src\input.cpp:21:

```

Reply:
Perfect, thanks for testing it out!